spec/features/idv/doc_auth/ssn_step_spec.rb

```diff
@@ -29,7 +29,7 @@
       fill_out_ssn_form_fail
       click_idv_continue
 
-      expect(page.find('#doc_auth_ssn')['aria-invalid']).to eq('value-missing')
+      expect(page.find('#doc_auth_ssn')['aria-invalid']).to eq('true')
```

Comment:
This will rely on #5667 to pass, which is currently held up by CodeClimate issues. I'll rebase once that's merged.